sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/ScriptTransformation.scala

```diff
@@ -25,13 +25,13 @@ import org.apache.spark.sql.catalyst.plans.ReferenceAllColumns
  *
  * @param script the command that should be executed.
  * @param output the attributes that are produced by the script.
- * @param ioschema the input and output schema applied in the execution of the script.
+ * @param ioSchema the input and output schema applied in the execution of the script.
```

Comment:
Let's avoid this kind of renaming that is virtually noop but just makes the conflicts when you backport other changes.

Reply:
You are right, I will close this PR.